master/buildbot/newsfragments/githuboauth2apiv4.bugfix

```diff
@@ -0,0 +1 @@
+Appended server URL with `/api/graphql` as a standard instead of `/graphql`
```

Comment:
I think the impact of the bugfix would be clearer if we said something like "Fixed Github v4 API URL"